app/Services/EntityTypeService.php

```diff
@@ -133,48 +133,13 @@ public function save(): EntityType
         $this->entityType->icon = $this->request->get('icon');
         $this->entityType->save();
 
-        if ($this->entityType->wasRecentlyCreated) {
-            $this->permissions()->bookmark();
```

Comment:
Why does it no longer create the permissions and the bookmark for the custom module?